lib/spack/spack/test/hg_fetch.py

```diff
@@ -15,16 +14,8 @@
 from spack.fetch_strategy import HgFetchStrategy
 from spack.spec import Spec
 from spack.stage import Stage
-from spack.util.executable import which
```

Comment:
This applies to the svn test as well. Does this function in envs w/o hg/svn? It seems like that skip should still be relevant here.

Reply:
It's not expressly obvious but there's still a skip applied in the test fixture `mock_hg_repository`

Reply:
Thanks! In that case this looks good. LGTM.